Mathlib/Data/Polynomial/Degree/Lemmas.lean

```diff
@@ -131,8 +130,8 @@ theorem natDegree_mul_C_eq_of_mul_eq_one {ai : R} (au : a * ai = 1) :
 set_option linter.uppercaseLean3 false in
 #align polynomial.nat_degree_mul_C_eq_of_mul_eq_one Polynomial.natDegree_mul_C_eq_of_mul_eq_one
 
-/-- Although not explicitly stated, the assumptions of lemma `nat_degree_mul_C_eq_of_mul_ne_zero`
-force the polynomial `p` to be non-zero, via `p.leading_coeff ≠ 0`.
+/-- Although not explicitly stated, the assumptions of lemma `natDegree_mul_C_eq_of_mul_ne_zero`
+force the polynomial `p` to be non-zero, via `p.leadingCoeff ≠ 0`.
```

Comment:
This PR is starting to do too many fixes at once. That makes it really hard to review. The diff is massive, so it should be extremely unsurprising. Please make changes like this in a different PR.

Reply:
Please review #5708.